File: cpp/6-2.cpp
#include "disp.h"//使用Display单元库
unsigned long audio1,audio2;
int main(){
audio1=loadaudio("display.mp3");//读取音频1
audio2=loadaudio("display.mp3");//读取音频2
playaudio(audio1);//播放音频1
msgbox("正在播放音频");
playaudio(audio2,true);//重复播放音频2
msgbox("正在重复播放音频");
return 0;}

File: pas/6-2.pas
uses Display;//使用Display单元库
var audio1,audio2:longword;
begin
audio1:=LoadAudio('display.mp3');//读取音频1
audio2:=LoadAudio('display.mp3');//读取音频2
PlayAudio(audio1);//播放音频1
Msgbox('正在播放音频');
PlayAudio(audio2,true);//重复播放音频2
Msgbox('正在重复播放音频');
end.
